test/14_voucherKernel.ts

```diff
@@ -660,5 +660,71 @@ describe('VOUCHER KERNEL', () => {
         )
       ).to.be.revertedWith(revertReasons.INVALID_QUANTITY_LONG);
     });
+
+    it('[createTokenSupplyId] Should not revert if validFrom is 5 minutes or more than ValidTo', async () => {
+      await deployContracts();
+
+      // spoof boson router address
+      await contractVoucherKernel.setBosonRouterAddress(users.deployer.address);
+
```

Comment:
Since 

      await deployContracts();
      // spoof boson router address
      await contractVoucherKernel.setBosonRouterAddress(users.deployer.address);
      
Repeates in last three tests, just put them togeether in describe and add that to beforeEach that does it.

Reply:
done